trillian/migrillian/core/controller.go

```diff
@@ -30,9 +31,32 @@ import (
 
 	"github.com/google/trillian/merkle"
 	_ "github.com/google/trillian/merkle/rfc6962" // Register hasher.
+	"github.com/google/trillian/monitoring"
 	"github.com/google/trillian/types"
 )
 
+type metrics struct {
+	masterships    monitoring.Counter
+	masterCancels  monitoring.Counter
+	isMaster       monitoring.Gauge
+	entriesFetched monitoring.Counter
+	entriesSeen    monitoring.Counter
+	entriesStored  monitoring.Counter
+	// TODO(pavelkalinnikov): Add latency histograms, latest STH, tree size, etc.
+}
+
+func newMetrics(mf monitoring.MetricFactory) metrics {
+	const treeID = "treeID"
+	return metrics{
+		masterships:    mf.NewCounter("masterships", "Number of mastership runs.", treeID),
```

Comment:
I think this name could be clearer. masterRuns?

Reply:
Done.